core/arch/arm/pta/sdp_pa.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <string.h>
+
+#include <sdp_pa.h>
```

Comment:
minor: shift above `#include <string.h>`

Reply:
Yes, will fix it in next patch.